src/test/data/perf_seeds/legacy_seed/expected_seed.json

```diff
@@ -0,0 +1,2556 @@
+{
```

Comment:
wdyt about re-building all `expected_seed.json` with `useProtoFieldName: true,` in this PR?
or do you want to commit this file as is and then rewrite it separately?

Reply:
updated test files